tests/e2e/config.json

```diff
@@ -0,0 +1,31 @@
+{
+  "src_folders": ["./tests/e2e/spec"],
+  "output_folder": "./tests/e2e/logs",
+  "custom_commands_path" : "node_modules/nightwatch-custom-commands-assertions/js/commands",
+  "custom_assertions_path" : "node_modules/nightwatch-custom-commands-assertions/js/assertions",
+  "page_objects_path": "",
+  "selenium": {
+    "start_process": true,
+    "server_path": "tests/e2e/bin/selenium.jar",
+    "log_path": "./tests/e2e/logs",
+    "host": "127.0.0.1",
+    "port": 4444,
+    "cli_args": {
+      "webdriver.chrome.driver": "tests/e2e/bin/chromedriver"
+    }
+  },
+  "test_settings": {
+    "default": {
+      "launch_url": "",
+      "silent": true,
+      "desiredCapabilities": {
+        "browserName": "chrome",
```

Comment:
I have written one Browserstack task as well (https://github.com/AsifAmin/js-testing-starter/blob/master/tests/e2e/conf.js). 

I think browserstack needs to be a standalone task because it takes a lot of time to run just few basic e2e tests and if it is there it will slow down the release process a lot. Also we never know when we discontinue the license.

My strategy is making Browserstack as standalone so people can run but that won't be part of the release process.